src/lib/redux/userData.ts

```diff
@@ -20,13 +20,39 @@ const refreshUserData = async (user: any): Promise<void> => {
   }
 };
 
+//Function is called at a set interval and repulls data from backend 
+// set interval is 15 min/900000 ms
+const refreshDataBackground = async (): Promise<void> => {
+  try {
+    const state = store.getState();
+    //need to refresh data in the back ground
+    if (state.userData.isOnline) {
```

Comment:
Doesn't this function get run even if we aren't logged in? We'll get an empty `userData` and probably error out in that case. We want to only call `refreshDataBackground` if the user is currently logged in.